schema/models_directory/pb/qm/dft/functional/components_directory/range_separated_exchange.json

```diff
@@ -0,0 +1,23 @@
+{
+    "schemaId": "models-directory-dft-definitions-components-range-separated-x",
```

Comment:
`schemaId`s for files in this directory should be adjusted to follow the directory name